app/javascript/multilang_en.css

```
@@ -0,0 +1,3 @@
+:root[lang="en"] .multilang:not([lang="en"]) {
+	display: none;
```

We don't have a linter for CSS yet, so that I would appreciate swapping the `tab` indention with 2 `spaces`.

```suggestion
  display: none;
```